### PR TITLE
update requests version for fixing vul item

### PR DESCRIPTION
**Pull request by lesscodingmorehappiness**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -27,6 +27,7 @@ dependencies = [
     "spurplus ~= 2.3.5",
     "websockets ~= 10.3",
     "charset_normalizer ~= 2.1.1",
+    "requests ~= 2.32.0",
```
Comment (squirrelsc) [inline pyproject.toml]: Which package referred it? There is one on `ado` part, please also add version for it.

Comment (lesscodingmorehappiness) [reply pyproject.toml]: it's referred by adal; and adal is referred by msrestazure.

added for the one in ado part

Comment (squirrelsc) [reply pyproject.toml]: `msrestazure` is in `azure`, so this should be added in the `azure` part.

**Review by squirrelsc — APPROVED** [review]

**Comment by squirrelsc** [comment]
@lesscodingmorehappiness please organize commits meaningful next time.

**Pull request merged**